### PR TITLE
Chance to win ui + Time in Queue

### PR DESCRIPTION
**Pull request by yonbergman**
solves #47 & #43 
tekken4

### CODE REVIEW

**Comment by udiWertheimer** [comment]
ole

**Comment by yonbergman** [comment]
@udiWertheimer @giladgo I updated the current match ui to get the margins and sizes perfect
screen shot 2015-03-11 at 11 10 08 am

File: app/src/main/java/com/iic/mokojin/PlayerQueueFragment.java [hunk 1]
```diff
@@ -119,6 +125,25 @@ public void onLoaded(List<QueueItem> queueItems, Exception e) {
                     mNonDismissedPositions = new ArrayList(Arrays.asList(positionsArray));
                 }
             });
+            
+            scheduleUpdateClock();
+        }
+
+
+        private void scheduleUpdateClock(){
```
Comment (giladgo) [inline app/src/main/java/com/iic/mokojin/PlayerQueueFragment.java]: Why are you notifying a change every second?

Comment (giladgo) [inline app/src/main/java/com/iic/mokojin/PlayerQueueFragment.java]: I think this should be in the activity, and it should do 
`mAdapter.notifyDataSetChanged()`.
This way you don't need to pass the activity to the adapter.

**Comment by giladgo** [comment]
:palm_tree:

**Pull request merged**